packages/augur-state/src/db/DB.ts

```diff
@@ -110,36 +110,31 @@ export class DB<TBigNumber> {
   public async sync(augur: Augur<TBigNumber>, chunkSize: number, blockstreamDelay: number): Promise<void> {
     let dbSyncPromises = [];
     const highestAvailableBlockNumber = await augur.provider.getBlockNumber();
+
+    for (let trackedUser of await this.trackedUsers.getUsers()) {
+      for (let userSpecificEvent of this.userSpecificEvents) {
+        let dbName = this.getDatabaseName(userSpecificEvent.name, trackedUser);
+          dbSyncPromises.push(
+            this.syncableDatabases[dbName].sync(
+              augur,
+              chunkSize,
+              blockstreamDelay,
+              highestAvailableBlockNumber
+            ));
+      }
+    }
+
     for (let dbIndex in this.syncableDatabases) {
       dbSyncPromises.push(
         this.syncableDatabases[dbIndex].sync(
           augur,
           chunkSize,
           blockstreamDelay,
           highestAvailableBlockNumber
-        )
-      );
+        ));
     }
 
-    for (let trackedUser of await this.trackedUsers.getUsers()) {
```

Comment:
`this.trackedUsers.getUsers()` can fail. If it does then the promises built from `syncableDatabases` will be unhandled. Also somehow this caused a subsequent call to `DB.sync` to fail.

Reply:
Good find! Thanks for fixing this.

Reply:
:D